lib/Thrift/Thrift.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) != 'Windows_NT'">netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup Condition="$(OS) == 'Windows_NT'">

```

Comment:
change it to 

```xml
  <ItemGroup Condition="'$(TargetFramework)' == 'net46'">
    <Reference Include="System.Net.Http" />
  </ItemGroup>
```

otherwise it bring .NET Core nuget package, while what we need is the lib that is part of installed .NET on 4.6